src/pack.rs

```diff
@@ -89,6 +90,31 @@ fn load_lockfile(manifest_path: &Path) -> Result<LockFile> {
     })
 }
 
+async fn build_local_package(
+    manifest_path: &Path,
+    output_dir: &Path,
+    platform: &str,
+) -> Result<()> {
+    let manifest_dir = manifest_path
+        .parent()
+        .ok_or_else(|| anyhow!("could not get parent directory of manifest_path"))?;
+    let status = TokioCommand::new("pixi")
+        .arg("build")
+        .arg("--output-dir")
+        .arg(output_dir)
+        .arg("--build-platform")
+        .arg(platform)
+        .current_dir(manifest_dir)
+        .stdout(std::process::Stdio::null())
+        .stderr(std::process::Stdio::null())
+        .status()
+        .await?;
```

Comment:
i think especially if we failed to build, it would be nice to see stdout/stderr (or stdout+stderr)

Also if we do `-v` I think it would be nice to show the output of pixi-build.